packages/cli/templates/js/src/main.ts

```diff
@@ -0,0 +1,206 @@
+import {
+  createFieldPlugin,
+  type FieldPluginResponse,
+  type FieldPluginData,
+  type FieldPluginActions,
+} from '@storyblok/field-plugin'
+import './style.css'
+
+type Component = {
+  html: (props: { data: FieldPluginData }) => string
+  setup: (props: { data: FieldPluginData; actions: FieldPluginActions }) => void
+  update: (props: { data: FieldPluginData }) => void
+}
+
+const rootElement = document.querySelector('#app')!
```

Comment:
@johannes-lindgren ah really? I thought we assumed `#app` exists. And the `/get_html` actually provides `#app` if I remember correctly. But if you think we should check and create, then I will apply it to other templates as well.

Reply:
I could also recall that /get_html would create #app, but now when I checked with this review, I can see that it just produces an empty div.

I thought it was best to avoid #app, as it might interfere with fieldtype-wrapper.js. But since it's not there, we can use #app.

But I think we should omit the div element from the templates, as the app won't mount in production unless they create it by code. It's tempting to clean up seemingly useless code. I guess we should also leave a comment, explaining why we create the element in such way.

Reply:
@johannes-lindgren sorry but I'm a bit confused. What do you propose about this?

Reply:
@eunjae-lee I'm referring [to this](https://github.com/storyblok/field-plugin/pull/176#discussion_r1185801292). The developer can feel tempted to remove the code that adds the element to the document, because in development mode, it does seemingly nothing. However, in production, the div element with #app/#plugin is not there anymore and the app fails to mount.

In addition to removing <div id="app"> from the document, we could also leave a comment to explain why we're adding the element at runtime.